samples/guide/src/main/java/okhttp3/recipes/kt/YubikeyClientAuth.kt

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.recipes.kt
+
+import java.io.IOException
+import java.security.KeyStore
+import java.security.SecureRandom
+import java.security.Security
+import java.util.Arrays
+import javax.net.ssl.KeyManagerFactory
+import javax.net.ssl.KeyStoreBuilderParameters
+import javax.net.ssl.SSLContext
+import javax.net.ssl.X509ExtendedKeyManager
+import javax.security.auth.callback.Callback
+import javax.security.auth.callback.CallbackHandler
+import javax.security.auth.callback.PasswordCallback
+import javax.security.auth.callback.UnsupportedCallbackException
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.internal.SuppressSignatureCheck
+import okhttp3.internal.platform.Platform
+
+object ConsoleCallbackHandler : CallbackHandler {
```

Comment:
Seems like this should be defined after `YubikeyClientAuth`, the class that names the file & has documentation?